ci/images/google-windows-geode-builder/packer.json

```diff
@@ -85,7 +85,7 @@
         "Move-Item \"C:\\Program Files\\BellSoft\\LibericaJDK-11*\" c:\\java11",
         "choco install -y liberica8jdk",
         "Move-Item \"C:\\Program Files\\BellSoft\\LibericaJDK-8*\" c:\\java8",
-        "choco install -y openssh --version 7.7.2.1 /SSHServerFeature",
+        "choco install openssh -params '/SSHServerFeature' -confirm",
```

Comment:
choco docs say `-y` or `--yes` or `--confirm` all mean the same thing, not sure about `-confirm`